server/webhook_http_test.go

```diff
@@ -600,6 +600,20 @@ func TestWebhookHTTP(t *testing.T) {
 			ExpectedIgnored: true,
 			CurrentInstance: false,
 		},
+		"CLOUD comment created - user mentioned": {
+			Request:                 testWebhookRequest("webhook-cloud-comment-created-mention-user.json"),
+			ExpectedSlackAttachment: true,
+			ExpectedHeadline:        "Test User **commented** on story [TES-41: Unit test summary 1](https://some-instance-test.atlassian.net/browse/TES-41)",
+			ExpectedText:            "> Added a comment with mentioned user [~test-mm-username]",
```

Comment:
I think we want this to be `@test-mm-username` right? To have a working MM user mention

Reply:
I don't want to block on the main purpose of this PR, but if we're going to change the mention functionality here, we should make it work as we intended in our PR discussion